src/pluto/helpers.py

```diff
@@ -134,12 +144,9 @@ def convert(dt):
 
 def blacklist_branches(branches):
     """For pluto this is done by programmatically, can also be hard-coded"""
-    rv = []
-    version_regex = r"[0-9]{2}v[0-9]"
+    valid_branches = []
     data_regex = r"[0-9]{4}-[0-9]{2}-[0-9]{2}"
```

Comment:
Incredibly minor and not introduced in your changes, but while we're here I'm guessing this is meant to be a "date_regex"?

Reply:
ah on line 148? yup that's worth fixing!